app/build.gradle.kts

```diff
@@ -101,6 +101,7 @@ dependencies {
     debugImplementation(libs.androidx.compose.ui.testManifest)
 
     implementation(libs.accompanist.systemuicontroller)
+    implementation(libs.androidx.activity.activity)
```

Comment:
I think this import should be unnecessary, since we already depend on `activity-compose` below. Can we avoid the new import?